practices_tasks/practices_tasks/Program.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace practices_tasks
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("As I understand it is better to user collections when you are not sure about the size of your massive.");
```

Comment:
good answer. logically it is correct. but what about the empty array? Is it possible?